lsp-lens.el

```diff
@@ -309,6 +309,13 @@ CALLBACK - callback for the lenses."
           (funcall callback lsp-lens--backend-cache lsp--cur-version)
         (lsp-lens--backend-fetch-missing lsp-lens--backend-cache callback lsp--cur-version)))))
 
+(defun lsp--lens-on-refresh (workspace)
+  "Clear lens within all buffers of WORKSPACE, refreshing all workspace buffers."
+  (cl-assert (not (eq nil workspace)))
+  (cl-loop
```

Comment:
nit: why using `cl-loop` instead of `mapc`?

Reply:
I just copied from the same did on semantic-tokens 😅 I don't like cl-loop either